notifier/src/main/kotlin/Notifier.kt

```diff
@@ -43,15 +43,20 @@ class Notifier(
 ) : ScriptRunner() {
     private val customProperties = buildMap {
         put("ortResult", ortResult)
-
-        config.mail?.let { put("mailClient", MailNotifier(it)) }
-        config.jira?.let { put("jiraClient", JiraNotifier(it)) }
-
+        put("mailClient", config.mail?.let { MailNotifier(it) })
```

Comment:
As the condition is now not at the outer level anymore, `buildMap` can be replaced with `mapOf` and `"mailClient" to config.mail?.let { MailNotifier(it) }` etc.